ios/SketchView/SketchViewContainer.m

```diff
@@ -41,11 +41,17 @@ -(SketchFile *)saveToLocalCache
 
 + (UIImage *) imageWithView:(UIView *)view
 {
-    UIGraphicsBeginImageContextWithOptions(view.bounds.size, view.opaque, [[UIScreen mainScreen] scale]);
+    UIGraphicsBeginImageContext(view.bounds.size);
```

Comment:
Suppose I could use `UIGraphicsBeginImageContextWithOptions` with the `opaque` flag set to false, or pass in the parameter like before.